oak-blob-cloud-azure/src/test/resources/azure.properties

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Microsoft Azure authentication credentials.
+# https://azure.microsoft.com/en-us/documentation/articles/storage-create-storage-account
+accountName=
+accountKey=
+
+# Microsoft Azure blob storage container name. Container is a grouping of a set
+# of blobs. https://msdn.microsoft.com/en-us/library/dd135715.aspx
+azureContainer=
+
+# The timeout interval, in milliseconds, to use for the request (default 5 minutes)
+requestTimeout=120000
+
+# Concurrent number of simultaneous requests per operation (default 1)
+concurrentRequestsPerOperation=3
+
+# Number of retries per request (default 3)
+maxRequestRetry=10
+
+# proxy configurations (optional)
+proxyHost=
+proxyPort=
```

Comment:
See my comment for constants. It will be good where ever we can to use the same user specific property